src/commands/tick-cancel-timer.js

```diff
@@ -0,0 +1,26 @@
+import db from '../db'
+import { writeRemoveTimer } from './output'
+
+export default { builder, handler: cancelTimer }
+
+function builder(yargs) {
+  return yargs
+  .usage('Usage: tick cancel-timer')
+  .example('tick cancel-timer', 'cancels the current timer without commiting')
+}
+
+function cancelTimer() {
+  db.get('_local/timers')
+  .then(removeTimer)
+  .then(writeRemoveTimer)
+  .catch(err => console.log(`Could not cancel your timer\n${err.message}`))
+}
+
+function removeTimer(timersDoc) {
+  const timer = timersDoc.timers.pop()
+
+  if (!timer) throw { message: 'You do not have a timer started' }
```

Comment:
Instead of throwing an object with a `message` property, it would be much more idiomatic to throw an `Error`. While this works as it is, any future code that calls `removeTimer` may want to call `.trace()` or other builtin functionality on the error.